puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/katello-rpm-pipeline.yaml

```diff
@@ -0,0 +1,25 @@
+- job-template:
+    name: 'katello-{version}-rpm-pipeline'
+    project-type: pipeline
+    sandbox: true
+    dsl:
+      !include-raw:
+        - 'pipelines/vars/katello/{version}.groovy'
+        - 'pipelines/release/katello.groovy{empty}'
+        - 'pipelines/lib/release.groovy{empty}'
+        - 'pipelines/lib/ansible.groovy{empty}'
+        - 'pipelines/lib/foreman_infra.groovy{empty}'
+        - 'pipelines/lib/rpm_packaging.groovy{empty}'
+        - 'pipelines/lib/obal.groovy{empty}'
+
+- project:
+    name: katello
+    jobs:
+      - 'katello-{version}-rpm-pipeline'
+    empty: ''
+    version:
+      - '3.10'
+      - '3.11'
```

Comment:
pretty sure we can drop these two (also from the vars)

Reply:
yeah, probably true.